examples/auto-scaling-groups/init.pp

```diff
@@ -0,0 +1,71 @@
+ec2_securitygroup { 'test-sg':
+  ensure      => present,
+  description => 'group for testing autoscaling group',
+  region      => 'sa-east-1',
+}
+
+ec2_launchconfiguration { 'test-lc':
+  ensure          => present,
+  security_groups => ['test-sg'],
+  region          => 'sa-east-1',
+  image_id        => 'ami-67a60d7a',
+  instance_type   => 't1.micro',
+}
+
+ec2_autoscalinggroup { 'test-asg':
+  ensure               => present,
+  min_size             => 2,
+  max_size             => 4,
+  region               => 'sa-east-1',
+  launch_configuration => 'test-lc',
+  availability_zones   => ['sa-east-1b', 'sa-east-1a'],
+}
+
+ec2_scalingpolicy { 'scaleout':
+  ensure             => present,
+  auto_scaling_group => 'test-asg',
+  scaling_adjustment => 30,
+  adjustment_type    => 'PercentChangeInCapacity',
```

Comment:
After going through the tutorial, it's not intuitive to me where this keyword comes from and what values could be put in here. I'd like to see this documented in the type parameter and perhaps validated with newvalues() if there are a limited number of allowed keywords here. Same with ChangeInCapacity in the scalin policy and the metric, statistic, comparison_operator, in the alarm resources.